modules/imapauth/default-disable

```diff
@@ -0,0 +1,3 @@
+This file indicates that the default state of this module
```

Comment:
If you make this a module installable with composer, I would recommend to remove this and add a `default-enable` file, as then modules will be enabled as soon as they are installed, minimizing the chances for misunderstandings or trouble.